Dependencies.props

```diff
@@ -5,25 +5,24 @@
     <Microsoft_AspNetCore_Http_Abstractions_Version>2.2.0</Microsoft_AspNetCore_Http_Abstractions_Version>
     <Microsoft_AspNetCore_Http_Version>2.2.2</Microsoft_AspNetCore_Http_Version>
     <Microsoft_Azure_DocumentDB_Version>2.11.0</Microsoft_Azure_DocumentDB_Version>
-    <Microsoft_Net_Test_Sdk_Version>16.6.1</Microsoft_Net_Test_Sdk_Version>
+    <Microsoft_Net_Test_Sdk_Version>16.7.1</Microsoft_Net_Test_Sdk_Version>
     <Microsoft_SourceLink_GitHub_Version>1.0.0</Microsoft_SourceLink_GitHub_Version>
     <Microsoft_SourceLink_AzureRepos_Git_Version>1.0.0</Microsoft_SourceLink_AzureRepos_Git_Version>
-    <MSTest_TestAdapter_Version>2.1.1</MSTest_TestAdapter_Version>
-    <MSTest_TestFramework_Version>2.1.1</MSTest_TestFramework_Version>
+    <MSTest_TestAdapter_Version>2.1.2</MSTest_TestAdapter_Version>
+    <MSTest_TestFramework_Version>2.1.2</MSTest_TestFramework_Version>
     <System_Configuration_ConfigurationManager_Version>4.7.0</System_Configuration_ConfigurationManager_Version>
     <System_ServiceModel_Primitives_Version>4.7.0</System_ServiceModel_Primitives_Version>
     <System_Threading_Tasks_Version>4.3.0</System_Threading_Tasks_Version>
-    <Xunit_Runner_VisualStudio_Version>2.4.2</Xunit_Runner_VisualStudio_Version>
+    <Xunit_Runner_VisualStudio_Version>2.4.3</Xunit_Runner_VisualStudio_Version>
+    <Moq_Version>4.14.5</Moq_Version>
```

Comment:
Consider sorting. I believe the list is otherwise sorted.

Reply:
Oh, missed that, thank you, sorted now